src/pages/Subscription/Subscription.jsx

```diff
@@ -1,9 +1,14 @@
+import { useState } from "react";
 import Button from "../../Components/Button/Button";
 import Input from "../../Components/Input/input";
 
 import "./Subscription.scss";
 
 export default function SubscriptionPage() {
+  const [inputState, setInputState] = useState("");
+  function handleButtonClick() {
+    window.alert(inputState);
+  }
   return (

```

Comment:
```suggestion
export default function SubscriptionPage() {
  const [inputState, setInputState] = useState("");
  
  function handleButtonClick() {
    window.alert(inputState);
  }
  
  return (
```

Add empty line after states and function